### DISCUSSION TITLE
Fritz3370 - Paket http_tunnel (hts) startet nicht?

### DISCUSSION THREAD

**Discussion by BeJoNe** [Freetz-NG/freetz-ng#196] [Allgemein] [answered]
Ich habe das paket http_tunnel (binary only) aktiviert.
Wenn ich per ssh auf die Box gehe und `hts --help` eingebe, bekomme ich die Hilfe angezeigt.
Sobald ich hts mit `hts -F localhost:22 8080` starte, beendet sich das programm sofort wieder.
Auch ein `ps | grep hts` zeigt nichts an?
Kann jemand helfen?

**Comment by fda77** [comment, marked as answer]
Installier noch das Package "strace" und start so: `strace -fff    hts -F localhost:22 8080`

**Reply by BeJoNe** [reply]
ok, interessanterweise nach strace funktionierts..
Danke

**Reply by fda77** [reply]
Das ist ja nur zum debuggen gedacht, um zu sehen wobei es crasht!
